components/views/auth/Register.tsx

```diff
@@ -21,11 +30,70 @@ const VALIDATIONS = {
 }
 
 const Register = () => {
-    const { form, handleChange } = useForm(INITIAL_STATE, VALIDATIONS)
+    const dispatch = useDispatch<AppDispatch>()
+    const router = useRouter()
+    const user = useSelector((state: any) => state.user)
+    const { form, handleChange, handleReset } = useForm(
+        INITIAL_STATE,
+        VALIDATIONS
+    )
+
     const { firstName, lastName, email, password } = form
     const [disableSubmit, setDisableSubmit] = useState<boolean>(true)
+
+    const [register] = useMutation(REGISTER, {
+        onCompleted({ register: data }) {
+            if (data.__typename === 'Errors') {
+                dispatch(
+                    showBanner({
+                        message: data.message,
+                        type: data.__typename
+                    })
+                )
+            } else {
+                const profile = { ...data }
+                const token = data.token
+                localStorage.setItem(JWT_SECRET, token)
+
+                delete profile.__typename
+                delete profile.successType
+                delete profile.token
+
+                dispatch(setProfile(profile))
+                dispatch(
+                    showBanner({
+                        message: 'User created in successfully',
+                        type: data.__typename
+                    })
+                )
+
+                handleReset()
+            }
+            dispatch(setLoading(false))
+            router.push('/dashboard')
+        },
+        onError(err: any) {
+            dispatch(setLoading(false))
+            dispatch(
+                showBanner({
```

Comment:
I think we can create a utility function of this in the banner slice, and export and use it.

```
showTechnicalDifficultiesBanner()

(slice file)
        showTechnicalDifficultiesBanner: (state) => {
            state.show = true
            state.message = 'We are experiencing technical difficulties. Please try again, or reach out for assistance at info@ezpzcoding.com'
            state.type = 'Errors'
        },
```

That way, we don't have to retype the error everywhere. Just import this in every file that will need the error catching